src/components/CryptCard/index.tsx

```diff
@@ -0,0 +1,62 @@
+import {StyleSheet, Text, View, Image} from 'react-native';
+import React, {useMemo} from 'react';
+import crypto from '../../../metrics.json';
+import PercentChangeArrow from '../../../assets/icons/north_east_black_24dp.svg';
+import {
+  ContainerView,
+  PercentChangeNumber,
+  PercentChangeView,
+  ColView,
+  LeftColTextView,
+  RightColTextView,
+  CryptImage,
+} from './styles';
+
+const CryptCard = () => {
+  const percentageChange = (percentNumber: number) => {
+    const roundNumber =
+      Math.round((percentNumber + Number.EPSILON) * 100) / 100;
+    const negative = roundNumber < 0;
```

Comment:
or both become an useMemo if you are using roundNumber in more places

Reply:
It says that I cannot use useMemo inside another useMemo for either 'negative' nor 'roundNumber' because "percentageChange" is already an useMemo. And I did the percentageChange useMemo as you instructed previously. So those values are already only to be re-calculated if the value passed as prop (percentNumber) changes.

Reply:
the error is because you can't use hooks outside the body of a component, you are inside a function in this case, a level inside the body

Reply:
![image](https://user-images.githubusercontent.com/20242580/230072219-b5167d8e-ffff-4eff-a802-00eab2d74acb.png)

you already have it declared outside, why don't you just move it to the top of the file and use it here to? so you will prevent redeclarations